libs/cardano-ledger-core/src/Cardano/Ledger/DPState.hs

```diff
@@ -136,26 +132,24 @@ instance NoThunks (InstantaneousRewards c) => NoThunks (DState c)
 instance NFData (InstantaneousRewards c) => NFData (DState c)
 
 instance (CC.Crypto c, ToCBOR (InstantaneousRewards c)) => ToCBOR (DState c) where
-  toCBOR (DState unified fgs gs ir ds) =
-    encodeListLen 5
+  toCBOR (DState unified fgs gs ir) =
+    encodeListLen 4
```

Comment:
:pray:

Reply:
Yeah .... , I didn't catch this one at first.